luigi/file.py

```diff
@@ -81,14 +81,16 @@ def __init__(self, path=None, format=None, is_tmp=False):
         self.format = format
         self.is_tmp = is_tmp
 
+    def mkdirs(self):
```

Comment:
I didn't know `os.makedirs` was a thing. But since it it? Why don't you call it `makedirs` then?